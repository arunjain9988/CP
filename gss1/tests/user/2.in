5
-1 -5 -3 0 8
6
1 2
2 3
3 5
1 5
4 4
4 5