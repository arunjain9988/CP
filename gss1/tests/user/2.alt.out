-1
-3
8
8
0
8
